install.sh

```diff
@@ -1,7 +1,78 @@
 #!/bin/bash
 
+#repo addresses
+aasdkRepo="https://github.com/OpenDsh/aasdk"
+gstreamerRepo="git://anongit.freedesktop.org/gstreamer/qt-gstreamer"
+openautoRepo="https://github.com/openDsh/openauto.git"
+
+#Help text
+display_help() {
+    echo
+    echo "   --deps           install all dependencies"
+    echo "   --aasdk          install and build aasdk"
+    echo "   --openauto       install and build openauto "
+    echo "   --gstreamer      install and build gstreamer "
+    echo "   --dash           install and build dash "
+    echo
+    exit 1
+}
+
+#location of OS details for linux
+OS_RELEASE_FILE="/etc/os-release"
+
+#check if Raspian is in the file, if not set the install Args to be false
+if grep -q "Raspbian" ${OS_RELEASE_FILE}; then
+  installArgs="-DRPI_BUILD=true"
+  isRpi=true
+else
+  installArgs=""
+  isRpi=false
+fi
+
+#check to see if there are any arguments supplied, if none are supplied run full install
+if [ $# -gt 0 ]; then
+  #initialize all arguments to false before looping to find which are available
+  deps=false
+  aasdk=false
+  gstreamer=false
+  openauto=false
+  dash=false
+  perms=false
+    while [ "$1" != "" ]; do
+        case $1 in
+            --deps )           shift
```

Comment:
spacing seems kinda wonky here? tbh not too familiar with switch cases in bash tho so excuse my ignorance if this is how it should be :p